Neos.Flow/Classes/Persistence/Doctrine/DataTypes/JsonArrayType.php

```diff
@@ -20,68 +25,38 @@
  *
  * @Flow\Proxy(false)
  */
-class JsonArrayType extends DoctrineJsonArrayType
+class JsonArrayType extends JsonType
 {
-    /**
-     * @var string
-     */
-    const FLOW_JSON_ARRAY = 'flow_json_array';
-
-    /**
-     * @var PersistenceManagerInterface
-     */
-    protected $persistenceManager;
+    private const FLOW_JSON_ARRAY = 'flow_json_array';
+    protected ?PersistenceManagerInterface $persistenceManager = null;
+    protected ReflectionService $reflectionService;
 
-    /**
-     * @var ReflectionService
-     */
-    protected $reflectionService;
-
-    /**
-     * Gets the name of this type.
-     *
-     * @return string
-     */
-    public function getName()
+    public function getName(): string
     {
         return self::FLOW_JSON_ARRAY;
     }
 
     /**
-     * Gets the (preferred) binding type for values of this type that
-     * can be used when binding parameters to prepared statements.
-     *
-     * @return integer
-     */
-    public function getBindingType()
-    {
-        return \PDO::PARAM_STR;
-    }
```

Comment:
Why has this been removed?

now it will return `ParameterType::STRING` due to the inheritance from `\Doctrine\DBAL\Types\Type::getBindingType`

Reply:
Because that actually is the same in `ParameterType`: `public const STRING = PDO::PARAM_STR;`